src/ioctls/vm.rs

```diff
@@ -83,15 +83,16 @@ impl VmFd {
     ///                     userspace_addr: 0x0 as u64,
     ///                     flags: 0,
     ///                 };
-    /// vm.set_user_memory_region(mem_region).unwrap();
+    /// unsafe {
+    ///     vm.set_user_memory_region(mem_region).unwrap();
+    /// };
     /// ```
     ///
-    pub fn set_user_memory_region(
+    pub unsafe fn set_user_memory_region(
```

Comment:
Whenever a method is declared as `unsafe`, we should add documentation stating what invariants must hold when calling the function such that the code is actually safe.